react-front-end/__tests__/tsrc/components/wizard/WizardHelper.test.ts

```diff
@@ -143,27 +142,68 @@ describe("render()", () => {
   });
 });
 
-describe("generateRawLuceneQuery()", () => {
-  jest
-    .spyOn(TokenisationModule, "getTokensForText")
-    .mockResolvedValue({ tokens: ["hello", "world"] });
+describe("generateAdvancedSearchCriteria()", () => {
+  it("builds raw Lucene query for Wizard controls", () => {
```

Comment:
Needs rewording... Perhaps:

```suggestion
  it("builds SearchAdditionalParams for Wizard controls", () => {
```

Reply:
oops I forgot to update the description. :smile: